tests/generation/Bears.py

```diff
@@ -357,3 +357,17 @@ def test_filter_bears_ci_mode(self):
                                     if bear not in res_1[lang]]
                 for bear in additional_bears_by_lang[lang]:
                     self.assertIn(bear, additional_bears)
+
+    def test_print_coala_quickstart_version_short(self):
+        sys.argv.append('-v')
+        with self.assertRaises(SystemExit) as cm:
+            with retrieve_stdout() as custom_stdout:
+                main()
+                self.assertEquals('0.4.0', custom_stdout.getvalue)
+
+    def test_print_coala_quickstart_version_long(self):
+        sys.argv.append('--version')
```

Comment:
Are the arguments reset on each test run? Not that you get something like `['-v', '--version']` because the other test function could execute earlier :)

Reply:
Yes, the setUp() and tearDown() function have done it